include/tpm-v2.h

```diff
@@ -660,6 +660,15 @@ u32 tpm2_report_state(struct udevice *dev, uint vendor_cmd, uint vendor_subcmd,
 u32 tpm2_enable_nvcommits(struct udevice *dev, uint vendor_cmd,
 			  uint vendor_subcmd);
 
+/**
```

Comment:
The commit message seems wrong. tcg2_get_pcr_info() returns both the active and supported banks